jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfigurationProducer.kt

```diff
@@ -60,7 +61,7 @@ class LocalLambdaRunConfigurationProducer : LazyRunConfigurationProducer<LocalLa
         val runtime = RuntimeGroup.determineRuntime(context.module) ?: RuntimeGroup.determineRuntime(context.project)
 
         setAccountOptions(configuration)
-        configuration.useHandler(runtime, handler)
+        configuration.useHandler(Runtime.fromValue(runtime.toString()), handler)
```

Comment:
Should this be .toSdkRuntime()?

Reply:
Ah nice catch, it should just be `runtime`. I converted this path to LambdaRuntime but we don't need to yet